repo/raja-perf/package.py

```diff
@@ -360,3 +360,7 @@ def initconfig_package_entries(self):
     def cmake_args(self):
         options = [f"-DMPI_CXX_LINK_FLAGS='{self.spec['mpi'].libs.ld_flags}'"]
         return options
+
+
+def _test_func():
```

Comment:
why is this here?

Reply:
That exists because I was using it to test the CI, and haven't got the CI working yet.